### PR TITLE
Quote FullyQualifiedName

### PR DESCRIPTION
**Pull request by Ghostbird**
Possible fix for #21. Please check whether this works.

### CODE REVIEW

**Comment by ahoischen** [comment]
This worked just fine on a simple F# test project, using Bash and xUnit. Powershell treats quotes mostly the same, so there should be no issues. However, `cmd` doesn't recognize these quotes so this won't work on default installations on win8.1 or lower.

I don't know how important `cmd` support is to this project, but I personally don't think supporting it is worth the effort. Single quotes are the simplest solution and work for almost all names (except those that contain single quotes), so this should be the accepted fix.

**Comment by Ghostbird** [comment]
Fix for #26 included.

**Comment by Ghostbird** [comment]
Ah, of course, this plugin is not limited to XUnit. Then it seems we need a different solution, since FullyQualifiedName does not seem to work (at least on my XUnit).

**Comment by Ghostbird** [comment]
Strange, I had to merge master into it twice to fix the conflicts.
Anyhow, I've removed the change that fixes #26, which broke things for non-XUnit frameworks.

So this still doesn't work with XUnit, but I'm unsure how to solve that. On my local machine I currently modify the JS files to work with XUnit.

File: src/dotnetTestExplorer.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ export class DotnetTestExplorer implements TreeDataProvider<TestNode> {
      * to do a restore, so it can be very slow.
      */
     public runTest(test: TestNode): void {
-        Executor.runInTerminal(`dotnet test${this.getDotNetTestOptions()}${this.outputTestResults()} --filter FullyQualifiedName~${test.fullName}`, this.testDirectoryPath);
+        Executor.runInTerminal(`dotnet test${this.checkBuildOption()}${this.checkRestoreOption()} --filter FullyQualifiedName~'${test.fullName}'`, this.testDirectoryPath);
```
Comment (formulahendry) [inline src/dotnetTestExplorer.ts]: You still not rebase correctly with master branch

Comment (formulahendry) [reply src/dotnetTestExplorer.ts]: For FullyQualifiedName, the format should be like https://stackoverflow.com/a/42787656/4545351. Single quote may not work on CMD of Windows.

**Pull request closed without merging**